aspnetcore/security/authentication/social/google-logins.md

```diff
@@ -4,17 +4,26 @@ author: rick-anderson
 description: This tutorial demonstrates the integration of Google account user authentication into an existing ASP.NET Core app.
 ms.author: riande
 ms.custom: "mvc, seodec18"
-ms.date: 06/19/2019
+ms.date: 10/08/2019
 uid: security/authentication/google-logins
 ---
 # Google external login setup in ASP.NET Core
 
 By [Valeriy Novytskyy](https://github.com/01binary) and [Rick Anderson](https://twitter.com/RickAndMSFT)
 
-[Legacy Google+ APIs have been shut down as of March 7, 2019](https://developers.google.com/+/api-shutdown). Google+ sign in and developers must move to a new Google sign in system. The ASP.NET Core 2.1 and 2.2 packages for Google Authentication have be updated to accommodate the changes. For more information and temporary mitigations for ASP.NET Core, see [this GitHub issue](https://github.com/aspnet/AspNetCore/issues/6486). This tutorial has been updated with the new setup process.
+[Legacy Google+ APIs have been shut down as of March 7, 2019](https://developers.google.com/+/api-shutdown). Google+ sign in and developers must move to a new Google sign in system. The ASP.NET Core 2.1 and 2.2 packages for Google Authentication have been updated to accommodate the changes. For more information and temporary mitigations for ASP.NET Core, see [this GitHub issue](https://github.com/aspnet/AspNetCore/issues/6486). This tutorial has been updated with the new setup process.
 
 This tutorial shows you how to enable users to sign in with their Google account using the ASP.NET Core 2.2 project created on the [previous page](xref:security/authentication/social/index).
 
+## Install the latest supporting NuGet package
```

Comment:
This is not correct for the doc. The preceding sentence is

This tutorial shows you how to enable users to sign in with their Google account using the ASP.NET Core 2.2 

You don't need to add the package to 2.2 projects - just 3.0 project.

Rather than adding a band-aid, let's update the topic for 3.0.

Reply:
@Rick-Anderson, thanks, I am trying to get familiar with the issue.  You want to keep the 2.2 version as it was and I should add a moniker for 3.0 correct?